src/strategies/php.ts

```diff
@@ -26,8 +26,9 @@ const CHANGELOG_SECTIONS = [
   {type: 'fix', section: 'Bug Fixes'},
   {type: 'perf', section: 'Performance Improvements'},
   {type: 'revert', section: 'Reverts'},
-  {type: 'chore', section: 'Miscellaneous Chores'},
-  {type: 'docs', section: 'Documentation', hidden: true},
+  {type: 'docs', section: 'Documentation'},
+  {type: 'misc', section: 'Miscellaneous'},
+  {type: 'chore', section: 'Chores', hidden: true},
```

Comment:
This is a breaking behavior change for other folks using the `php` strategy and this could be considered for a new semver major version.

Reply:
Makes sense. I just added `misc`, is something that is considered breaking?

Reply:
@chingor13 would it be possible to merge / release this before Friday? The second part of this (updating the `version` field in `composer.json` for the monorepo) is a blocked. I can split it into a separate PR if you'd like!

Reply:
Please separate the `version` field change into a separate PR.

Yes, adding `misc` will cause any `misc: ` commits to start forcing a patch bump when it is not part of the spec by default. These sections are already configurable in your repository without affecting every other user of that strategy.

Reply:
Yes, I understand. We just have a LOT of repositories and it'd be nice to be able to configure this in one place.